openapi-schema-oneof-referenced-schema.json

```diff
@@ -25795,7 +25795,7 @@
           "PubCalculations"
         ],
         "summary": "Get all calculations",
-        "description": "This endpoints returns all active calculation requests.",
+        "description": "This endpoints returns all calculation requests.",
```

Comment:
Didn't we deploy the GetAllCalculations end point changes earlier?

Reply:
Description changes were not release previously.